### ISSUE TITLE
Question about released model

### ISSUE THREAD

**Issue by cru-tak** [auspicious3000/contentvec#32]
Hello, Thank for your great work.
I have a question about the released models (e.g. checkpoint_best_legacy_500.pt,  checkpoint_best_100.pt).

Were these models finetuned with Librispeech 960h with labels? Or not?

Sorry for the basic question.
Best regards.

**Comment by auspicious3000** [comment]
These are pretrained model.

**Comment by cru-tak** [comment]
Understood. Thank you for your quick response.
